### PR TITLE
Add CHANGELOG items for many new changes since 0.8

### PR DESCRIPTION
**Pull request by huonw**
This was written by browsing the `git log`. I've collapsed all the CI/Github/devops changes into a brief section because I don't think they're relevant/interesting to most users.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,20 +6,24 @@
 
 ### Major features and improvements
 
+- StellarGraph is now available as [a conda package on Anaconda Cloud](https://anaconda.org/stellargraph/stellargraph) [\#516](https://github.com/stellargraph/stellargraph/pull/516)
 - New algorithms:
   - Cluster-GCN: an extension of GCN that can be trained using SGD, with demo [\#487](https://github.com/stellargraph/stellargraph/issues/487)
+  - `FullBatchLinkGenerator` has been added to do link prediction using full-batch algorithms like GCN, GAT, APPNP and PPNP [\#543](https://github.com/stellargraph/stellargraph/pull/543)
```
Comment (huonw) [inline CHANGELOG.md]: I'm not sure if we should consider this a "new algorithm" or not.

Comment (PantelisElinas) [reply CHANGELOG.md]: If you rephrase it to say something like "Link prediction for full batch models. FullBatchLinkGenerator..." then it will fit under the new algorithms banner.

Comment (huonw) [reply CHANGELOG.md]:
```suggestion
  - Link prediction for full-batch models: `FullBatchLinkGenerator` allows doing link prediction with algorithms like GCN, GAT, APPNP and PPNP [\#543](https://github.com/stellargraph/stellargraph/pull/543)
```
